docs/src/_reference/command-line-interface.md

```diff
@@ -87,7 +87,7 @@ meltano add extractor tap-ga--client-foo --inherit-from tap-google-analytics
 
 - `--variant=<variant>`: Add a specific (non-default) [variant](/concepts/plugins#variants) of the identified [discoverable plugin](/concepts/plugins#discoverable-plugins).
 
-- `--include-related`: Also add transform, dashboard, and model plugins related to the identified discoverable extractor.
+- `--include-related`: Also add transform plugins related to the identified discoverable extractor.
 
```

Comment:
@edgarrmondragon - Does your latest update honor `--include-related` when considering `requires`? 

If not, I think we should just remove this:

```suggestion
```

If so, I would reword:

```suggestion
- `--include-related`: Adds related plugins if they are required and not yet added to the project.

```